LESS/cta.less

```diff
@@ -4,38 +4,56 @@
 
     @media @desktop {
         order: 1;
+        width: 40%;
+        height: 30rem;
     }
 }
 
 .cta-main {
-    h1 {
-        font-family: @rway;
-        .header-imp;
-        margin: 1rem 0 .2rem 0;
-    }
-    h2 {
-        font-family: @rway;
-        margin: .2rem 0;
-    }
-    p {
-        font-family: @ssp;
-        margin: 1rem 0;
-        padding: 0 1.4rem 0 0;
-    }
-    .give {
-        .custom-btn(black; @mint);
-    }
-    
-    .receive {
-        .custom-btn(black; @white);
+
+    .cta-content {
+
+        @media @desktop {
+            margin-top: 3rem;
+            display: flex;
+            width: 32rem;
+        }
+
+        .content-container {
+            h1 {
+                font-family: @rway;
+                .header-imp;
+                margin: 1rem 0 .2rem 0;
+            }
+            h2 {
+                font-family: @rway;
+                margin: .2rem 0;
+            }
+            p {
+                font-family: @ssp;
+                margin: 1rem 0;
+                padding: 0 1.4rem 0 0;
+            }
+            .give {
+                .custom-btn(black; @mint);
+            }
+            
+            .receive {
+                .custom-btn(black; @white);
+            }
+        }
     }
+
     .banner {
+        font-family: @pf;
         text-align: center;
         margin: 2rem 0;
+        font-size: 1.8rem;
 
         @media @desktop {
+            margin-top: 5rem;
             text-align: left;
-            font-size: 1.5rem;
+            font-size: 3.4rem;
```

Comment:
https://github.com/eztip/eztip-UI-joe-pak/blob/613cf4e4d13b2212e23488eed45fc07487008d60/LESS/cta.less#L1-L59

Intelligent use of LESS again.